test/run-mocha.js

```diff
@@ -1,10 +0,0 @@
-/**
```

Comment:
We should check with @umurkontaci if we can remove that file. I guess we need to keep it, because it hooks `before` and `after` function calls which contain logic essential for some tests to work properly.